src/DisplayPanel/DisplayPanel.ts

```diff
@@ -82,7 +82,7 @@ class DisplayPanel implements DisplayInterface {
    * Update SVG based on visualization settings
    */
   updateVisualization (): void {
-    DisplayControls.setOpacityFromSlider(this.className);
+    // DisplayControls.setOpacityFromSlider(this.className);
```

Comment:
You can just delete lines you comment out - that's the beauty of version control!

Reply:
Was my plan but was convinced otherwise, we'll take a look at the opacity inheritance issue and remove the commented code